j2v/generation/processor.py

```diff
@@ -3,20 +3,23 @@
 from j2v.generation.generator import Generator
 from j2v.generation.result_writer import SQLWriter, LookerWriter
 from j2v.utils.config import generator_config
+from six import string_types
```

Comment:
Why do we need to include this package? Do we need to target python2 string_types? I think we can only focus on python3

Reply:
I added the depencency by mistake. I am going to remove it now.